src/ImportDefinitionsBundle/Setter/KeySetter.php

```diff
@@ -30,7 +30,7 @@ public function set(Concrete $object, $value, Mapping $map, $data)
         $setter = sprintf('set%s', ucfirst($setter[0]));
 
         if (method_exists($object, $setter)) {
-            $object->$setter(File::getValidFilename($value));
+            $object->$setter(DataObject\Service::getValidKey($value,"object"));
```

Comment:
Shouldn't be actually part of this PR, but we can make an exception and leave that.

Reply:
@dpfaffenbauer so could it be in this PR or not?

Reply:
yes, we can keep it. But usually you should create a separate one for stuff like that to keep PR's clean.

Reply:
I will be more careful next time